core/jvm/src/main/scala/cats/effect/IORuntimePlatform.scala

```diff
@@ -21,17 +21,19 @@ import scala.concurrent.duration._
 
 import java.util.concurrent.{CountDownLatch, TimeUnit}
 
-private[effect] abstract class IOPlatform[+A] { self: IO[A] =>
+private[effect] abstract class IORuntimePlatform { self: IORuntime =>
```

Comment:
The downside of this change is now `unsafeRunSync` is no longer available as a method *on* `IO`. I guess maybe that's okay though, since it's just a convenience function, so it kind of makes sense for it to have defaulted schedulers.

Reply:
Yeah, I've wanted `unsafeRun*` removed from `IO` for a few years now -- as a method, it wasn't enough of a speed bump to slow folks down. It requiring an `IORuntime` argument is a bit better but I'm afraid auto-complete will lead folks to `ioa.unsafeRunSync(IORuntime.default)`. Moving them off `IO` and to `IORuntime` is just enough friction to make a difference without unduly inconveniencing folks who